README.md

```diff
@@ -17,7 +17,7 @@ A list of example projects, useful to learn from and for bootstrapping new docum
 <!-- description -->
 
 Read the Docs is a fully open-source platform that builds and publishes documentation. Read more
-on [readthedocs.org](https://readthedocs.org/) and [readthedocs.com](https://readthedocs.com/) (Read the Docs for Business).
+on [readthedocs.org](https://readthedocs.org/) and [readthedocs.com](https://readthedocs.com/) (Read the Docs for Business). Documentation projects have traditionally been part of a software project. However, many new documentation projects have emerged in science and academia. Therefore, we are compiling a list of inspirational uses of documentation technology, especially outside of the traditional field of software documentation. It's a work in progress, please feel invited to [contribute](#contributing)!
```

Comment:
A reminder to not put everything in one line. It's hard to comment on a specific paragraph.

> Therefore, we are compiling a list of inspirational uses of documentation technology, especially outside of the traditional field of software documentation.

Is that the use of this page? I thought it was for all awesome docs projects we host?

Reply:
Two lines worked, linter is appeased

> Is that the use of this page? I thought it was for all awesome docs projects we host?

Can you elaborate the comment? In my mind the use of "especially" means that we are highlighting a special aspect of why this list is motivated, but the general "awesome" aspect remains.

Reply:
Split in several lines, linter ate it. Improvements welcome!